src/middlewares/one-of.ts

```diff
@@ -9,16 +9,22 @@ const dummyItem: ContextItem = { async run() {} };
 
 export type OneOfCustomMessageBuilder = (options: { req: Request }) => any;
 
+export type OneOfErrorType = 'grouped' | 'leastErroredOnly' | 'flat';
+
 export function oneOf(
   chains: (ValidationChain | ValidationChain[])[],
-  message?: OneOfCustomMessageBuilder,
+  options: { message: OneOfCustomMessageBuilder; errorType?: OneOfErrorType },
 ): Middleware & { run: (req: Request) => Promise<void> };
 export function oneOf(
   chains: (ValidationChain | ValidationChain[])[],
-  message?: any,
+  options: { message?: any; errorType?: OneOfErrorType },
 ): Middleware & { run: (req: Request) => Promise<void> };
```

Comment:
Hoping that in the future typescript will help us to avoid these awful signatures.